.github/workflows/release-dry-run.yml

```diff
@@ -25,8 +25,6 @@ jobs:
     runs-on: macOS-11
     steps:
       - uses: actions/checkout@v1
-      - name: Select Xcode
-        run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
```

Comment:
Since the `bundle`/`release` commands now switch Xcodes for themselves, we shouldn't need to select them here in the workflow file.